src/test/regress/expected/sequential_modifications.out

```diff
@@ -491,6 +490,142 @@ SELECT pg_reload_conf();
  t
 (1 row)
 
+-- The following tests are added to test if create_distributed_table honors sequential mode
```

Comment:
Why don't we have any tests that'd fail with `cannot distribute relations with data...`

We should have the following test:
```SQL
CREATE TABLE test_create_seq_table (a int);
INSERT INTO test_create_seq_table VALUES (1);

SET citus.multi_shard_modify_mode TO 'sequential';
SELECT create_distributed_table('test_create_seq_table' ,'a');

RESET citus.multi_shard_modify_mode;

BEGIN;
	SET LOCAL citus.multi_shard_modify_mode TO 'sequential';
	select create_distributed_table('test_create_seq_table' ,'a');
ROLLBACK;
```